Algorithm/QCAlgorithm.cs

```diff
@@ -877,8 +877,14 @@ public void SetBrokerageModel(IBrokerageModel model)
                 // update models on securities added earlier (before SetBrokerageModel is called)
                 foreach (var security in Securities.Values)
                 {
+                    // save the leverage specified in AddSecurity, just in case the SecurityInitializer overwrites it
+                    var leverage = security.Leverage;
```

Comment:
A kitten dies every time we merge a hack.. ideally we should detect that AddSecurity leverage has been set and only save the leverage off if its changed. How feasible is that without breaking everyone's algorithms :)?